mcwb/types.py

```diff
@@ -33,6 +34,36 @@ class Anchor(Enum):
     MIDDLE = CENTER = CENTRE = 'middle'
 
 
+class Anchor3(Enum):
+    """Anchor point for cuboids."""
+
+    MIDDLE = CENTER = CENTRE = "middle"
```

Comment:
Please prefer single quotes for regular strings.

Reply:
done in 00775b6

It is really helpful for collaboration to have a strict formatter that enforces rules like this in the CI. black is good for this (but uses double quotes for strings - you have to surrender any opinions about formatting to adopt it :-) )